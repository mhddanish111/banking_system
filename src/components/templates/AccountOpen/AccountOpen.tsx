import React from 'react';
import Layout from '../Layout';
import AccountForm from '../../organisms/AccountForm';

const AccountOpen: React.FC<any> = () => {
  return (
    <Layout pageHeading="Open Account">
      <AccountForm type="Accounts" />
    </Layout>
  );
};

export default AccountOpen;
