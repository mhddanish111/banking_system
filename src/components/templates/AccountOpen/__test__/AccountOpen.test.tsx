import { shallow } from 'enzyme';
import AccountOpen from '../AccountOpen';

describe('<AccountOpen />', () => {
  const wrapped = shallow(<AccountOpen />);
  it('should render the AccountOpen Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
