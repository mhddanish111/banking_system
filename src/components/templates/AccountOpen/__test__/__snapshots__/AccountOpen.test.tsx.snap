// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountOpen /> should render the AccountOpen Component correctly 1`] = `
<Layout
  pageHeading="Open Account"
>
  <AccountOpenForm
    type="Accounts"
  />
</Layout>
`;
